pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -60,6 +60,14 @@
             System.ComponentModel.Composition;
         </Value>
     </ValidatePackageSuppression>
+    <!-- Permit inbox revsion of assemblies that are part of shared framework but also ship as packages. This will
+    make sure other packages that have a reference to them will carry a package reference to the latest serviced
+    version. -->
+    <ValidatePackageSuppression Include="PermitInbox">
```

Comment:
Why was this required? Can you please share the error message you were getting?

Reply:
The private transport package was failing due to a version inconsistency, System.Text.Json was expected to be inbox with version 4.0.1.0 but was found to have 4.0.1.1. Here is the error message I got:

```
C:\Users\joperezr\.nuget\packages\microsoft.dotnet.build.tasks.packaging\1.0.0-beta.20113.5\build\Packaging.targets(1161,5): error : File System.Text.Json, version 4.0.1.1 was included framework package Microsoft.Private.CoreFx.NETCoreApp/4.7.0-dev.20167.1 but that version is not considered inbox in package index F:\git\corefx\pkg\Microsoft.Private.PackageBaseline\packageIndex.json.  Please add it with appropriate InboxOn entry for netcoreapp3.1 or suppress this message with PermitInbox suppression. [F:\git\corefx\pkg\Microsoft.Private.CoreFx.NETCoreApp\Microsoft.Private.CoreFx.NETCoreApp.pkgproj]
```